### PR TITLE
UCP: Fixes in wireup auxiliary selection.

### PR DESCRIPTION
**Pull request by yosefe**

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/4255/ for details (Mellanox internal link).

**Comment by yosefe** [comment]
@evgeny-leksikov pls take a look

File: src/ucp/wireup/select.c [hunk 1]
```diff
@@ -948,18 +957,21 @@ static double ucp_wireup_aux_score_func(ucp_context_h context,
 {
     /* best end-to-end latency and larger bcopy size */
     return (1e-3 / (ucp_wireup_tl_iface_latency(context, iface_attr, remote_iface_attr) +
-            iface_attr->overhead + remote_iface_attr->overhead)) +
-           (1e3 * ucs_max(iface_attr->cap.am.max_bcopy, iface_attr->cap.am.max_short));
```
Comment (alinask) [inline src/ucp/wireup/select.c]: Can you please explain why am_bcopy is removed?

Comment (yosefe) [inline src/ucp/wireup/select.c]: after this change, wireup transport will be selected only according to latency, and not the maximal size of am_bcopy. which i think should have been done in the first place.

**Pull request merged**